spec/item_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'item'
+
+describe Item do
```

Comment:
This class is mostly just storing things, not really having any behaviour. I wouldn't test it unless it had behaviour.

You could add a behaviour of storing numbers but having a display method that displays the nice formatted prices, and then that would be a behaviour to test for.